logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/ConversationEventReceiver.kt

```diff
@@ -197,6 +201,7 @@ class ConversationEventReceiverImpl(
                     content = message.content
                 )
             }
+            is MessageContent.TextEdited -> editTextHandler.handle(message,message.content)
```

Comment:
Awesome way of doing this. 👏🏼

I think this is something we can start doing for other message types. Having specialized classes for each thing.